.vscode/settings.json

```
@@ -1,3 +1,4 @@
 {
-  "editor.defaultFormatter": "biomejs.biome"
-}
+  "editor.defaultFormatter": "biomejs.biome",
+  "editor.formatOnSave": true
```

Just a suggestion, but BiomeJs has a quickfix setting as well which I believe gives you autofixing of lint issues as well, not just formatting :)
```suggestion
  "editor.formatOnSave": true,
  "editor.codeActionsOnSave": {
    "quickfix.biome": true
  }
```
ref. https://biomejs.dev/reference/vscode/#fix-on-save

---

Might aswell try it :)

---

https://github.com/biomejs/biome/issues/1631

I'll hold off on this until they get it working with VSCode 1.83+